components/application-registry/internal/metadata/servicedefservice_test.go

```diff
@@ -1,6 +1,10 @@
 package metadata
 
 import (
+	v1a "github.com/kyma-project/kyma/components/application-operator/pkg/apis/applicationconnector/v1alpha1"
```

Comment:
We can add an additional test case in case in case getting Application UID fails.

Reply:
Done!